meta/main.yml

```diff
@@ -3,7 +3,7 @@ galaxy_info:
   author: 'Brian Akins, Dustin Brown & Datadog'
   description: Install Datadog agent and configure checks
   license: Apache2
-  min_ansible_version: 1.6
+  min_ansible_version: 2.0
```

Comment:
Is there a way we could avoid increasing the minimum ansible version? Increasing it would require a major version bump on this role IMO.